Assets/Mirror/Runtime/LocalClient.cs

```diff
@@ -59,7 +59,7 @@ internal void AddLocalPlayer(NetworkIdentity localPlayer)
             ClientScene.InternalAddPlayer(localPlayer);
         }
 
-        void PostInternalMessage(short msgType, byte[] content)
+        void PostInternalMessage(int msgType, byte[] content)
```

Comment:
shouldn't this be marked as [obsolete] too?

Reply:
It is private,  nobody can use it,  and it is used internally by the new api